app/routes/user/email/index.tsx

```diff
@@ -56,6 +56,16 @@ export default function Index() {
           Add
         </Link>
       </div>
+      <div>
+        This page is for managing your GCN Email subscriptions. You can create
+        as many alerts as you like! Each alert will have a corresponding Name,
+        Email (the recipient), notice type, and a list of selectable notices.
+        The Email field will default to the one used to sign in, but can be
+        changed. To create a new alert, click the "Add" button above. Once an
```

Comment:
Avoid spatial descriptions of UI elements such as "above" because they may not have any meaning for vision-impaired users or users on screen readers.